relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -303,6 +312,28 @@ fn sanitized_span_description(
     Some(sanitized)
 }
 
+/// Trims the given span description to ensure
+/// [`relay-metrics::AggregatorService`] doesn't drop it for being too long.
+///
+/// If the description is short, it remains unchanged. If it's long, this method
+/// truncates it to the maximum allowed size and sets the last character to
+/// `*`.
+fn truncate_span_description(mut description: String, max_bytes: usize) -> String {
+    if bytecount::num_chars(description.as_bytes()) < max_bytes {
```

Comment:
can't you just do description.len() ? that will return the length of the string in bytes

https://doc.rust-lang.org/std/string/struct.String.html#method.len

Reply:
oops, didn't see jorris had already commented